File: namsor/kotlin/src/main/kotlin/oseg/namsor_examples/OriginExample.kt
package oseg.namsor_examples

import app.namsor.client.infrastructure.*
import app.namsor.client.apis.*
import app.namsor.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class OriginExample
{
    fun origin()
    {
        ApiClient.apiKey["X-API-KEY"] = "YOUR_API_KEY"

        try
        {
            val response = PersonalApi().origin(
                firstName = "Keith",
                lastName = "Haring",
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling PersonalApi#origin")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling PersonalApi#origin")
            e.printStackTrace()
        }
    }
}

File: namsor/typescript-axios/src/OriginExample.ts
import axios, { AxiosError } from "axios";
import * as api from "namsor_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.PersonalApi(configuration).origin(
  "Keith", // firstName
  "Haring", // lastName
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling PersonalApi#origin:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
